templates/master/00-master/_base/files/usr-local-bin-openshift-recovery-tools-sh.yaml

```diff
@@ -303,7 +303,7 @@ contents:
         exit 1
       fi
 
-      ID=$($ETCDCTL_WITH_TLS member list | awk -F "," "/\s${NAME}\,/"'{print $1}') || true
+      ID=$($ETCDCTL_WITH_TLS member list | grep "$NAME" | cut -d "," -f 1) || true
```

Comment:
You need the `\s` and `,` around `$NAME` to match properly, otherwise you'll match extra members (e.g. `NAME="infra1"` would match  `infra1` plus `infra10`,`infra11`..., `not_infra1`, etc)

Reply:
@ericavonb thanks for the notes, will add if we are still keeping the scripts in MCO